documentation/prettier/readme.md

```diff
@@ -0,0 +1,57 @@
+# Prettier Config Documentation
+
+> Documentation for the `@strv/prettier-config` module.
+
+## Usage
+
+### Installing
+
+Run:
+
+```bash
+# Yarn:
+yarn add --dev @strv/prettier-config
+
+# npm:
+npm install --save-dev @strv/prettier-config
+```
+
+And reference it in your `package.json` file:
```

Comment:
Not sure if I am consistent myself in this regard, but I tend to use _italics_ for file/path references. Could you please update all such situations to use italics instead of fixed-width fonts? 🙏

Reply:
Sure!